pages/articles/authentication.md

```diff
@@ -155,3 +155,44 @@ Permission | Permission type | Id | Type
 Connect-PnPOnline [yourtenant].sharepoint.com -Interactive -ClientId [clientid] -Tenant [yourtenant].onmicrosoft.com -AzureEnvironment [USGovernment|USGovernmentHigh|USGovernmentDoD|Germany|China]
 ```
 The AzureEnvironment parameter only allows one value. Select the correct one that matches your cloud deployment.
+
+## Silent Authentication with Credentials for running in Pipelines
+
+For running `Connect-PnPOnline` with user credentials in Azure DevOps pipeline, you need to make sure that authentication in your Azure AD application is configured to allow public client. 
+
+Public client can be configured from the Azure portal from the Authentication Blade in the application or by setting the `allowPublicClient` property in the application's manifest to true.
```

Comment:
Can you also please add a screenshot for this ?

Reply:
Done. Thanks for the review.

Reply:
@kzkalra  - these images need to be in PnP PowerShell repo.

Can you please add them in `pages > images` and create a new folder there and then reference them in docs:

<img width="268" alt="image" src="https://github.com/pnp/powershell/assets/8832018/2c62c989-64cb-47af-86ab-350b53d2cb87">

Reply:
Thanks for the review @gautamdsheth . I have made the suggested changes.

Reply:
@gautamdsheth Requesting for re-review. :)